package.json

```diff
@@ -69,6 +69,7 @@
     "@azure/core-client": "^1.0.0",
     "@azure/core-rest-pipeline": "^1.9.2",
     "@azure/identity": "^3.1.3",
+    "@azure/monitor-opentelemetry": "file:azure-monitor-opentelemetry-0.0.0-beta.0.tgz",
```

Comment:
Was this just being used for testing? If so can we remove it?

Reply:
Package doesn't exist in npm, that is why this PR is still a draft, will update when available